pkg/controller/webhooks.go

```diff
@@ -170,7 +170,7 @@ func (c *Controller) triggerRefMetricsPull(ctx context.Context, ref schemas.Ref)
 				}
 
 				if matches {
-					c.ScheduleTask(context.TODO(), schemas.TaskTypePullProject, ref.Project.Name)
+					c.ScheduleTask(context.TODO(), schemas.TaskTypePullProject, ref.Project.Name, ref.Project.Name, w.Pull)
```

Comment:
```suggestion
					c.ScheduleTask(context.TODO(), schemas.TaskTypePullProject, ref.Project.Name, ref.Project.Name)
```

Reply:
indeed, great catch, thanks! I feel the only bit missing is adding the project name a second time as an argument as the first one is used as a task ID. Would you mind reverting the other changes which I believe do not make any difference?

Reply:
without the wildcards pull config getting passed to a default config is generated which does not contain the users config for that wildcard

Before changing this I didn't have any merge request refs even though I had them configured in the wildcard AND in the project_defaults

Reply:
My bad, makes sense thanks!

Reply:
no worries! 
Thank you for creating the exporter!